packages/test/src/load/starter.ts

```diff
@@ -126,22 +127,33 @@ async function main() {
   const taskQueue = getRequired(args, '--task-queue');
 
   const connection = new Connection({ address: serverAddress });
-
   const client = new WorkflowClient(connection.service, { namespace });
-
   const stopCondition = runForSeconds ? new UntilSecondsElapsed(runForSeconds) : new NumberOfWorkflows(iterations);
 
-  const passed = await runWorkflows({
-    client,
-    workflowName,
-    taskQueue,
-    stopCondition,
-    concurrency: concurrentWFClients,
-    minWFPS,
-  });
-  if (!passed) {
-    console.error('Load test did not pass');
-    process.exit(1);
+  let workflowsToRun: string[];
+  if (workflowName === 'yummy-sampler-mode') {
```

Comment:
Why have this magic string instead of passing this in as multiple args?

Reply:
We can put it in another script, I don't like this special case.
Leaving it up to you to decide.

Reply:
challenge accepted ;-)
Okay, I won't press this